deb/publish.go

```diff
@@ -452,6 +453,36 @@ func (p *PublishedRepo) GetLabel() string {
 	return p.Label
 }
 
+// GetAppStream returns full path list of appstream files
+func (p *PublishedRepo) GetAppStream(component string) ([]string, error) {
+	if utils.Config.AppStreamDir == "" {
+		return []string{}, nil
+	}
+
+	appStreamDir := filepath.Join(utils.Config.AppStreamDir, p.Prefix, "dists", p.Distribution, component)
+	files := []string{}
+
+	err := filepath.Walk(appStreamDir, func (path string, info os.FileInfo, err error) error {
+		ext := filepath.Ext(path)
```

Comment:
small comment, you could move this line below the error check so as it's only needed when the file exists